src/core/main.go

```diff
@@ -161,8 +162,22 @@ func main() {
 
 	server.RegisterRoutes()
 
-	log.Infof("Version: %s, Git commit: %s", version.ReleaseVersion, version.GitCommit)
+	if common_http.InternalTLSEnabled() {
+		log.Info("internal TLS enabled, Init TLS ...")
+		iTLSKeyPath := os.Getenv("INTERNAL_TLS_KEY_PATH")
+		iTLSCertPath := os.Getenv("INTERNAL_TLS_CERT_PATH")
+
+		log.Infof("load client key: %s client cert: %s", iTLSKeyPath, iTLSCertPath)
+		// uncomment following if harbor2 is ready
```

Comment:
confusing comment

Reply:
just a note. because if we can enable mTLS in for if it's ready.

Reply:
I think we can remove it for now.  Otherwise please update the comment to clarify.  I don't think `if harbor2 is ready` is a clear criteria...

Reply:
ok let's remove them